perl.html.markdown

```diff
@@ -126,18 +126,20 @@ while (condition) {
 
 
 # for loops and iteration
-for (my $i = 0; $i < $max; $i++) {
+for my $i (0 .. $max) {
   print "index is $i";
 }
 
-for (my $i = 0; $i < @elements; $i++) {
+while (my $i = each @elements) {
```

Comment:
It isn't really idiomatic, but rather accomplishes the same thing. Honestly, I wouldn't really know of any reason to iterate over an array in this way to begin with. Might make sense just to drop this entirely.